### DISCUSSION TITLE
How do you add a policy to the default event bus in an account?

### DISCUSSION THREAD

**Discussion by nbertagnolli** [aws/aws-cdk#26948] [Q&A]
I want to setup an organization level policy so that I can receive events in one account from another account in my organization.  To do that my default event bus needs the policy:

```
  {
    "Version": "2012-10-17",
    "Statement": [
   			  
      {
        "Sid": "AllowAllAccountsFromOrganizationToPutEvents",
        "Effect": "Allow",
        "Principal": "*",
        "Action": "events:PutEvents",
        "Resource":  "arn:aws:events:us-west-2:1234565:event-bus/default",
        "Condition": {
          "StringEquals": {
            "aws:PrincipalOrgID": "o-....."
          }
        }
      }]
  }  
```

I can get the interface for the default eventbus with this:

```
default_event_bus = events.EventBus.from_event_bus_name(self, "default-bus", "default")
```

But I can't seem to attach a policy to this IEventBus. Does anyone know how to get this policy attached to the default eventbus? Sorry if this is a silly question but I'm stuck

**Comment by peterwoodworth** [comment]
Does [`EventBusPolicy`](https://docs.aws.amazon.com/cdk/api/v2/docs/aws-cdk-lib.aws_events.EventBusPolicy.html) work?
